stellar/operators.js

```diff
@@ -10,20 +10,57 @@ async function processCommand(options, _, chain) {
 
     const contract = new Contract(options.address || chain.contracts?.axelar_operators?.address);
 
-    let operation;
+    let operator, operation;
 
-    const address = Address.fromString(options.args || wallet.publicKey());
+    if (['is_operator', 'add_operator', 'remove_operator'].includes(options.action)) {
+        if (!options.args) throw new Error(`Missing --args operatorAddress the params.`);
+        operator = Address.fromString(options.args).toScVal();
+    } else {
+        operator = Address.fromString(wallet.publicKey()).toScVal();
+    }
 
     switch (options.action) {
         case 'is_operator':
-            operation = contract.call('is_operator', address.toScVal());
+            operation = contract.call('is_operator', operator);
             break;
         case 'add_operator':
-            operation = contract.call('add_operator', address.toScVal());
+            operation = contract.call('add_operator', operator);
             break;
         case 'remove_operator':
-            operation = contract.call('remove_operator', address.toScVal());
+            operation = contract.call('remove_operator', operator);
+            break;
+
+        case 'refund': {
+            // eslint-disable-next-line no-case-declarations
+            const gasService = options.target || chain.contracts?.GasService?.address;
+            if (!gasService) throw new Error(`Missing AxelarGasService address in the chain info.`);
+
+            const target = Address.fromString(gasService).toScVal();
+            const method = nativeToScVal('refund', { type: 'symbol' });
+            const [messageId, receiver, tokenAddress, tokenAmount] = parseArgs(options.args || '');
+            const args = nativeToScVal([
+                messageId,
+                Address.fromString(receiver),
+                { address: Address.fromString(tokenAddress), amount: tokenAmount },
+            ]);
+
+            operation = contract.call('execute', operator, target, method, args);
+            break;
+        }
+
+        case 'execute': {
+            if (!options.target) throw new Error(`Missing target address param.`);
+            const target = Address.fromString(options.target).toScVal();
+
+            if (!options.method) throw new Error(`Missing method name param.`);
+            const method = nativeToScVal(options.method, { type: 'symbol' });
+
+            const args = nativeToScVal(parseArgs(options.args || ''));
```

Comment:
if args is empty, this should be converted to an empty Vec ScVal

Reply:
`nativeToScVal` does that